### ISSUE TITLE
Make the /path command in game a private message

### ISSUE THREAD

**Issue by CardSwim8** [Hanabi-Live/hanabi-live#2951]
Avoid implied table talk by making the /path command and response private

**Comment by Zamiell** [comment]
do u wanna submit a pr?

**Comment by CardSwim8** [comment]
i wish. way outside my skillset :(

**Comment by rz-1** [comment]
I disagree, I think the /path command should stay global b/c it's an easy way to share the doc with newer players.

There is already a way to accomplish this: go to lobby > resources button > Strategies and Conventions url.
If you want to make a separate command that whispers the link to the user that seems fine.

**Comment by CardSwim8** [comment]
i feel like we could have our cake and eat it to on this one: make it public in the general chat and in the pre-game chat, but private during live games? probably harder to implement though

**Comment by rz-1** [comment]
Personally, I'd still disagree. 
But it's probably feasible to add a parameter to the /path command. Something like:
/path @msg_location <"global"/"msg">

Now whether that's worth the time to develop when there's a known workaround is another story.

**Comment by Zamiell** [comment]
> I disagree, I think the /path command should stay global b/c it's an easy way to share the doc with newer players.

i also kind of like it being global because, in the situation where you have 3 max level players playing with 1 level 6 player, now ALL three max level players will have to type /path instead of just one having to type path and then the other two simply clicking on the link, if that makes sense.

**Comment by CardSwim8** [comment]
i can see reasoning for that. maybe then rz's original suggestion of a /pm_path command or something similar so that people aren't inadvertently influencing others by using the /path command either before giving a clue or immediately after receiving one

**Comment by Zamiell** [comment]
> it's an easy way to share the doc with newer players

well you are supposed to use /doc for that, not /path

**Comment by rz-1** [comment]
> well you are supposed to use /doc for that, not /path

then I guess I'm confused, whats the purpose of /path when you can use /doc and then click on "learning path" from the main doc page?

**Comment by Zamiell** [comment]
> then I guess I'm confused, whats the purpose of /path when you can use /doc and then click on "learning path" from the main doc page?

to prevent having the extra step of having to click on "learning path" from the main doc page

**Comment by Zamiell** [comment]
> This looks bloat to me

yeah having both /doc and /path is probably bloat, but i am fairly lazy in not wanting to click an additional time, so I like it, for now anyway
